CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Change Log
+All notable changes to QingStor SDK for JAVA will be documented in this file.
+
+## [v2.1.2] - 2017-01-13
+
+### Changed
+
+- Fix ：Object name is contain the same as the bucket name will get error suffix path.
+- Reference document modification.
+- Add CHANGLELOG and Installation guide.
+
+
+## [v2.1.1] - 2016-12-28
+
+### Changed
+
+- Android adapter.
+
+
+## [v2.1.0] - 2016-12-25
+
+### Changed
+
+- Fix signer bug.
+- Add more parameters to sign.
+
+### Added
```

Comment:
For a better looking, we make the label organized as alphabet order.
The first should be `Added`, than `Changed`, than `Fixed`, and so on.
The only exception is that `BREAKING CHANGE` shoud be placed at bottem.